python/src/puml/gen_puml.aac

```diff
@@ -0,0 +1,131 @@
+            then:
+              - A PlantUML component diagram of the model is printed to the console.
+          - name: Output PlantUML component diagram to file.
+            given:
+              - The {{puml.input.architecture-file}} contains a valid model.
+              - The positional argument {{puml.input.output-directory}} contains a valid filepath.
+            when:
+              - The aac app is run with the puml-component command.
+            then:
+              - A PlantUML component diagram of the model is generated to a .puml file.
+    - name: puml-sequence
+      help_text: Converts an AaC-defined usecase to PlantUML sequence diagram.
+      input:
+        - name: architecture-file
+          type: file
+          description: |
+            A path to a YAML file containing an AaC-defined usecase from which
+            to generate a PlantUML sequence diagram.
+        - name: output-directory
+          type: directory
+          description: |
+            The output directory into which the PlantUML (.puml) diagram file
+            will be written.
+      acceptance:
+        name: Sequence Diagram Test
+        scenarios:
+          - name: Output PlantUML sequence diagram for valid use case.
+            given:
+              - The {{puml.input.architecture-file}} contains a valid model.
+            when:
+              - The aac app is run with the puml-sequence command.
+            then:
+              - A PlantUML sequence diagram of the use case is printed to the console.
+          - name: Output PlantUML sequence diagram to file.
+            given:
+              - The {{puml.input.architecture-file}} contains a valid model.
+              - The positional argument {{puml.input.output-directory}} contains a valid filepath.
+            when:
+              - The aac app is run with the puml-sequence command.
+            then:
+              - A PlantUML sequence diagram of the model is generated to a .puml file.
+    - name: puml-object
+      help_text: Convert an AaC-defined system to PlantUML object diagram.
+      input:
+        - name: architecture-file
+          type: file
+          description: |
+            A path to a YAML file containing an AaC-defined system from which to
+            generate a PlantUML object diagram.
+        - name: output-directory
+          type: directory
+          description: |
+            The output directory into which the PlantUML (.puml) diagram file
+            will be written.
+      acceptance:
+        name: Object Diagram Test
+        scenarios:
+          - name: Output PlantUML object diagram for valid model.
+            given:
+              - The {{puml.input.architecture-file}} contains a valid model.
+            when:
+              - The aac app is run with the puml-object command.
+            then:
+              - A PlantUML object diagram of the model is printed to the console.
```

Comment:
```suggestion
```

Looks like these are left over from initial work on the plugin. According to JD it started as dump to console then moved to file output. Console was dropped at some point, but definition didn't get updated.

I just tested legacy to see if it would dump to console if I did not provide an output directory and get the following return:

```
(venv) gitpod /workspace/AaC/python/model/alarm_clock (puml-changes-requirements-diagram) $ aac puml-object alarm_clock.yaml 
Usage: aac puml-object [OPTIONS] ARCHITECTURE_FILE OUTPUT_DIRECTORY
Try 'aac puml-object -h' for help.

Error: Missing argument 'OUTPUT_DIRECTORY'.
```

Reply:
Changed to "written to the output directory".